src/base.js

```diff
@@ -396,7 +404,7 @@ export default class KintoClientBase {
         method: request.method,
         headers: request.headers,
         body: stringify ? JSON.stringify(request.body) : request.body,
-      },
+      }),
```

Comment:
Sorry about that! I probably would have chosen something more like this:

```
{
    ...selectProperties(request, ["method", "headers"]),
    body: stringify ? JSON.stringify(request.body) : request.body,
}
```

with

```
function selectProperties(obj, props) {
    let ret = {};
    for (const prop of props) {
        if (obj.hasOwnProperty(prop)) {
            ret[prop] = obj[prop];
        }
    }
    return ret;
}
```

But I guess this is fine too.